### PR TITLE
Adding Pod name for idle email notification

### PR DESCRIPTION
**Pull request by fbm3307**
Adding Pod name for idle email notification 
Related to https://github.com/codeready-toolchain/member-operator/pull/478
and issue https://issues.redhat.com/browse/SANDBOX-186

### CODE REVIEW

File: deploy/templates/notificationtemplates/sandbox/idlertriggered/notification.html
Comment (mfrancisc) [inline deploy/templates/notificationtemplates/sandbox/idlertriggered/notification.html:50]: just a small reminder - based on https://github.com/codeready-toolchain/member-operator/pull/478#discussion_r1346958235 should we rename the variable to `.AppName` ?

Comment (fbm3307) [reply deploy/templates/notificationtemplates/sandbox/idlertriggered/notification.html:50]: will be changing it applicationname

**Comment by fbm3307** [comment]
Updated the PR

File: deploy/templates/notificationtemplates/appstudio/idlertriggered/notification.html
Comment (alexeykazakov) [inline deploy/templates/notificationtemplates/appstudio/idlertriggered/notification.html:49]: Let's use CammelCase as we mentioned in the other PR.

Comment (fbm3307) [reply deploy/templates/notificationtemplates/appstudio/idlertriggered/notification.html]: Done

**Comment by mfrancisc** [comment]
Not related to this change - but since this change depends on the member operator change this means both PRs need to be merged and deploy at the same time right? 

If that is true then could we move those idle notification assets to member operator (if it's the one that is using them) ? And make this process simpler.

**Comment by mfrancisc** [comment]
@fbm3307 I think you also need to update the test , please check : https://github.com/codeready-toolchain/host-operator/actions/runs/6466713541/job/17555291564?pr=878

**Comment by fbm3307** [comment]
> @fbm3307 I think you also need to update the test , please check : https://github.com/codeready-toolchain/host-operator/actions/runs/6466713541/job/17555291564?pr=878

Updated it

**Comment by alexeykazakov** [comment]
> Not related to this change - but since this change depends on the member operator change this means both PRs need to be merged and deploy at the same time right?

Not really. The member operator change should go first. It just populate the new template params which wouldn't be used in the currently deployed host operator templates (from the master branch). So the templates won't be broken.
Then when the member operator PR is merged and deployed we can merge this host PR which introduces new templates which would now use the params populated by the member operator.

**Comment by mfrancisc** [comment]
> Not really. The member operator change should go first.

Got it, the template will not be broken as soon as the member operator PR goes in for first. Thanks for explaining that 👍 

I had a quick chat with @MatousJobanek , and he also explained me some of the historical reasons and told me that there are plans for decoupling more the html templates from the operators. 

One quick idea could be to add the html inside the Notification CR, something like:
```
apiVersion: toolchain.dev.openshift.com/v1alpha1
kind: Notification
.....
spec:
  context:
    emailBodyHTML: <p>Yehaw!</p>
```
so that host-operator responsibility is just to send/manage those notifications, and html templates can be stored outside (in member-operator or any other external operator that wants to create a Notification CR ) .

**Review by MatousJobanek — COMMENTED** [review]
Looks all good, let's wait until the other PR in member operator is approved and merged

**Comment by MatousJobanek** [comment]
> One quick idea could be to add the html inside the Notification CR, something like

We already support that for custom email notifications (that have content that cannot be specified as a template - like the ToolchainStatus one). 
But for these cases, we specify only the template name because adding the html template in a spec of the notification CR is a duplication of the same thing that is already present in the host cluster. 
However, we could change the notification controller so it updates the status with the actual content of what was sent via Mailgun - that would make testing and investigation of any issues much easier. But this is something to be done as a separate task of course :-) I created an epic for the whole effort of customizing the email templates  [SANDBOX-433](https://issues.redhat.com/browse/SANDBOX-433) It could be done as part of this effort

**Review by alexeykazakov — APPROVED** [review]

**Pull request merged**
